client/src/partials/reports/baseReports.service.js

```diff
@@ -63,15 +46,14 @@ function BaseReportService($http, Modal, util) {
    * Formats a reports configuration options with the PDF report API and returns
    * a request for the PDF document generation.
    */
-  function requestPDF(url, report, reportOptions) {
-    var pdfParams = {
+  function saveReport(url, report, reportOptions) {
```

Comment:
In the future, we should probably standardize this API to have both methods (`requestPreview` and `saveReport` take `reportId` instead of `report.id` and `reportId`.